charts/workspacemanager/templates/deployment.yaml

```diff
@@ -87,7 +89,37 @@ spec:
         - name: cloudsql-sa-creds
           mountPath: /secrets/cloudsql
           readOnly: true
+      {{- if .Values.enableProxy }}
+      - name: oidc-proxy
+        image: "{{ .Values.proxyImage.repository }}:{{ .Values.proxyImage.version }}"
+        ports:
+        - containerPort: 443
+        - containerPort: 80
```

Comment:
Is there a reason to even expose port 80 when we have the proxy set up? Don't we want all traffic to be https then?

Reply:
I put that in because it seemed nicer to leave 80 open and having the proxy redirect to https instead of just timing out connections to 80. Not a strong reason, so happy to change this if that's a bad practice

Reply:
Listening on port 80 is a best practice, as long as you send an appropriate redirect to https. @zloery's response is spot-on:

> I put that in because it seemed nicer to leave 80 open and have the proxy redirect to https instead of just timing out connections to 80.

ETA: This can be handled upstream in the connection chain (at an LB or other ingress point), but it's good behavior to handle common errors like something using http in a URL.

Reply:
I see now, got it!